addons/rating/static/src/models/message.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import { addFields, patchModelMethods } from '@mail/model/model_core';
+import { attr } from '@mail/model/model_field';
+// ensure the model definition is loaded before the patch
+import '@mail/models/message';
+
+patchModelMethods('Message', {
+    /**
+     * @override
+     */
+    convertData(data) {
+        const data2 = this._super(data);
+        if ('rating_image_url' in data) {
+            data2.ratingImgUrl = data.rating_image_url;
+        }
+        return data2;
+    },
+
+});
+
+addFields('Message', {
+    /**
+     * The source url of the rating value image displayed
+     */
+    ratingImgUrl: attr(),
```

Comment:
Also PR is not introducing the rating model in JS? It's not completely necessary now, but it might be limiting us later (same problem as we had for tracking).

Reply:
no, should I do that first? or merge this first and then introduce that later.

Reply:
If we do it it should be now, otherwise we just don't do it until we need it more.

Reply:
then I would suggest not doing it now. since the rating part is not really our part and needs more review from their team? Unlike mail_tracking_value

Reply:
Well it's trivial to just create the rating model in js and have id value and image url fields, instead of storing them on message.

Reply:
ok, then I will quickly do it!